### DISCUSSION TITLE
how to install garudo on a chromebook

### DISCUSSION THREAD

**Discussion by zakaria0002** [dnschneid/crouton#4683] [Q&A]
hello
how can i  install garudo linux on a chromebook

**Comment by foxsouns** [comment]
i assume you mean garuda linux, and the way that i know of would be to use the mrchromebox script to enable legacy boot for your chromebook, make the garuda usb drive, and then use it to erase the chrome os drive and install garuda. this only works if your chromebook is x86_64, and you meet the minimum requirements for garuda. i would highly recommend instead using either debian or xubuntu, as chromebooks are typically underpowered and xfce works well on low ram.

**Reply by zakaria0002** [reply]
thank you for your reply
Is it possible to install it with crouton?

**Reply by foxsouns** [reply]
probably not, as crouton uses the kernel from chrome os, which is often well behind the latest bleeding edge, and also nobody has programmed it in yet, so you would have to implement it yourself. debian is there as an option, though! try using bullseye on crouton, and buster if you run into any problems. the readme.md is really helpful for installs via crouton, give it a read.
